doc/source/contributing_code.rst

```diff
@@ -687,7 +687,7 @@ Then, generate the docs with:
 
     .. code-block:: bash
 
-        python setup.py build_sphinx -E
+        cd doc/sphinx && make html
```

Comment:
So may a controversial opinion, but if we're going to eventually encourage folks to run doctest on the regular, would it make sense here to use what we do in CI and call `sphinx-build`?

Maybe I don't understand the advantage of calling `make html` over `sphinx-build` though.

Reply:
I find it simpler than `sphinx-build` since it reads the source and destination files from the Makefile, and it's an [endorsed way](https://www.sphinx-doc.org/en/master/usage/quickstart.html#running-the-build) to build docs. I'm not too attached either way.

Reply:
Can we run doctests through the make call? If not, I'd prefer stick to sphinx-build.

Reply:
Yup, `make doctest` -- it's in the Makefile.